templates/Uwp/Testing/UnitTests.App.MSTest/Param_ProjectName.Tests.MSTest/Param_ProjectName.Tests.MSTest.csproj

```diff
@@ -125,13 +125,27 @@
   </ItemGroup>
   <ItemGroup>
     <Content Include="Properties\UnitTestApp.rd.xml" />
-    <Content Include="Assets\LockScreenLogo.scale-200.png" />
-    <Content Include="Assets\SplashScreen.scale-200.png" />
-    <Content Include="Assets\Square150x150Logo.scale-200.png" />
-    <Content Include="Assets\Square44x44Logo.scale-200.png" />
-    <Content Include="Assets\Square44x44Logo.targetsize-24_altform-unplated.png" />
-    <Content Include="Assets\StoreLogo.png" />
-    <Content Include="Assets\Wide310x150Logo.scale-200.png" />
```

Comment:
Shouldn't these seven items be removed since you're replacing them each with a content item that uses a link?

Reply:
@mslukewest, can't see whats wrong here, could you give more details?

Reply:
Ignore my comment, I read the diff wrong and thought all the items had duplicates.  I see now that they don't.